evm_arithmetization/src/generation/jumpdest.rs

```diff
@@ -0,0 +1,191 @@
+use std::cmp::max;
+use std::{
+    collections::{BTreeSet, HashMap},
+    fmt::Display,
+    ops::{Deref, DerefMut},
+};
+
+use itertools::{sorted, Itertools};
+use keccak_hash::H256;
+use serde::{Deserialize, Serialize};
+
+/// Each `CodeAddress` can be called one or more times, each time creating a new
+/// `Context`. Each `Context` will contain one or more offsets of `JUMPDEST`.
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize, Default)]
+pub struct ContextJumpDests(pub HashMap<usize, BTreeSet<usize>>);
```

Comment:
An odd name. Shall we keep it as `Context` so other code can use it as `jumpdest::Context`?